.github/workflows/webpack-jumpstart.yml

```diff
@@ -0,0 +1,80 @@
+name: webpack-jumpstart
+on:
+  pull_request_target:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: none
+    timeout-minutes: 20
+    env:
+      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+    steps:
+      - name: Build jumpstart tarball
+        run: |
+          id="$(podman container create -u node -w /home/node node /bin/sh -c \
+            'git init cockpit &&
+             cd cockpit &&
+             git fetch --depth 1 https://github.com/cockpit-project/cockpit '${MERGE_SHA}':build &&
+             git checkout build &&
+             pkg/build webpack-jumpstart.tar -j$(nproc)'
+          )"
+
+          podman container start --attach "${id}" >&2
+          podman container cp "${id}":cockpit/webpack-jumpstart.tar .
+          podman container rm -f "${id}"
+
+      - name: Create artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: webpack-jumpstart
+          path: webpack-jumpstart.tar
+          retention-days: 1
+
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: none
+    environmnent: cockpit-dist upload
+    timeout-minutes: 5
+    env:
+      GIT_DIR: git-dir.git
+      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+    steps:
+      - name: Set up work area
+        run: |
+            set -ux
+
+            git init -b main
+
+            git config user.name "GitHub Workflow"
+            git config user.email "cockpituous@cockpit-project.org"
+
+            git remote add cache "ssh://git@github.com/${GITHUB_REPOSITORY%/*}/cockpit-dist"
+
+      - uses: actions/download-artifact@v2
```

Comment:
Nice that there's an official one now. You figure we don't need to pin this down to a vetted SHA any more?

Related to that, as further fortification we could [change out action settings](https://github.com/cockpit-project/cockpit/settings/actions) to only allow `actions/*` and `octokit/*` actions, so that it becomes harder to slurp in something untrusted.

Reply:
We have a habit of using freely from the `actions/` namespace without pinning.

This "official one" has existed forever.  We couldn't use it before, though, because it only supports downloading an artifact from the same workflow.